File: MobileUnified/react-native/src/mobileGameState.ts
export type MobileGameState = {
  mode: string;
  cash: number;
  level: number;
  score: number;
  bizVirtual: boolean;
  network: 'offline' | 'online';
};

export const startMobileGame = (mode = 'default', cash = 10000): MobileGameState => ({
  mode,
  cash,
  level: 1,
  score: 0,
  bizVirtual: true,
  network: 'offline',
});

File: MobileUnified/android/app/src/main/java/io/amerhwitat/bizx/mobile/MobileGameState.kt
package io.amerhwitat.bizx.mobile

data class MobileGameState(
    val mode: String = "default",
    val cash: Long = 10_000,
    val level: Int = 1,
    val score: Long = 0,
    val bizVirtual: Boolean = true,
    val network: String = "offline"
)

object MobileGameEngine {
    fun start(mode: String = "default", cash: Long = 10_000) = MobileGameState(mode, cash)
}
